gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-debug/src/main/java/io/gravitee/gateway/debug/reactor/processor/DebugEventCompletionProcessor.java

```diff
@@ -87,9 +87,19 @@ public void handle(ExecutionContext context) {
         response.statusCode(debugContext.response().status());
         response.setBody(((VertxHttpServerResponseDebugDecorator) debugContext.response()).getBuffer().toString());
         debugApi.setResponse(response);
+        computeInvokerResponse(debugContext, debugApi);
         return debugApi;
     }
 
+    private void computeInvokerResponse(DebugExecutionContext debugContext, io.gravitee.definition.model.debug.DebugApi debugApi) {
+        HttpResponse invokerResponse = new HttpResponse();
+        Map<String, List<String>> headers = convertHeaders(debugContext.getInvokerResponse().getHeaders());
+        invokerResponse.setHeaders(headers);
+        invokerResponse.statusCode(debugContext.getInvokerResponse().getStatus());
+        invokerResponse.setBody(debugContext.getInvokerResponse().getBuffer().toString());
+        debugApi.setBackendResponse(invokerResponse);
+    }
```

Comment:
I think it would be more clear to return the invokerResponse here and to the set on L90